vitrina/datasets/forms.py

```diff
@@ -362,6 +362,56 @@ def __init__(self, request=None, organization=None, *args, **kwargs):
         )
 
 
+class InformationSystemResourceForm(BaseResourceForm):
+    class Meta:
+        model = Dataset
+        fields = (
+            "title",
+            "description",
+            "is_public",
+            "tags",
+            "catalog",
+            "frequency",
+            "access_rights",
+            "files",
+            "name",
+            "contact",
+            "creator",
+            "publisher",
+            "managed_by_publisher",
+            "landing_page",
+            "information_system_type",
+        )
+
+    def __init__(self, request=None, organization=None, *args, **kwargs):
+        super().__init__(request, organization, *args, **kwargs)
+
+        self.helper.layout = Layout(
+            Field("is_public", placeholder=_("Ar duomenys vieši")),
```

Comment:
"Ar" makes it a question -.-

```suggestion
            Field("is_public", placeholder=_("Ar duomenys vieši?")),
```

Reply:
Fixed in 3ea040ee4611a47b23f6dc6472a957502c116590